targets/LuaSdk/make.js

```diff
@@ -176,6 +176,10 @@ function getResultAction(tabbing, apiCall) {
     var internalTabbing = tabbing + "    ";
     if (apiCall.url === "/Authentication/GetEntityToken")
         preCallback = internalTabbing + "PlayFabSettings._internalSettings.entityToken = result.EntityToken\n";
+    if (apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
```

Comment:
A couple of issues with this addition:
1. I don't think it makes sense to store the game server token in a separate setting. part of the point of making this change is that, after authenticating, future calls automatically include the new token in the auth header.
2. You clear it immediately after setting it.